crates/typst/src/export/render.rs

```diff
@@ -407,7 +397,9 @@ fn render_outline_glyph(
         // Premultiply the text color.
         let Paint::Solid(color) = text.fill;
         let c = color.to_rgba();
-        let color = sk::ColorU8::from_rgba(c.r, c.g, c.b, 255).premultiply().get();
+        let color = sk::ColorU8::from_rgba(c.r, c.g, c.b, 255).premultiply();
+        let color =
+            u32::from_ne_bytes([color.red(), color.green(), color.blue(), color.alpha()]);
```

Comment:
We should probably just use PremultipliedU8 instead of doing this. But I wasn't sure how to change the `alpha_mul` and `blend_src_over` method.

Reply:
Perhaps we can also use `bytemuck` here to have the cast be fast.

Reply:
Is this how you meant it? Sorry, I've never used `bytemuck`. 😅